app/search_builders/curation_concerns/collection_search_builder.rb

```diff
@@ -8,27 +8,31 @@ module CurationConcerns
   #
   class CollectionSearchBuilder < ::SearchBuilder
     include FilterByType
+
     # Defines which search_params_logic should be used when searching for Collections
-    def initialize(context)
+    # @param [#blacklight_config, #current_ability] scope the object that has access to #blacklight_config
+    #   From a controller, scope would be `self`.  This argument is passed to ::SearchBuilder.new()
+    # @param [Symbol] permissions defining breadth of search, e.g. :edit, :read
+    # @note permissions will otherwise be defaulted by inherited #discovery_permissions
+    def initialize(scope, access = nil)
       @rows = 100
-      super(context)
+      @discovery_permissions ||= access_levels[access] if access
```

Comment:
Not necessarily.  If this class is extended and *that* class' initializer sets the @discovery_permissions before calling this one, then we keep those (e.g. bypassing `access_levels`).  But you are right we can probably do without it.  

You can mostly ignore this PR while it is still WIP.